src/components/Spinner/index.js

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+import './styles.css';
+
+const Spinner = () => (
+  <div className="sk-circle">
+    <div className="sk-circle1 sk-child" />
```

Comment:
may be this better `{[...Array(11)].map((item, index) => <div className={`sk-circle${index + 1} sk-child`} />)}`

Reply:
@gianglevan94 :\